.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 
       conda config --set show_channel_urls true
       conda update --yes conda
-      conda install --yes conda-build=1.20.0 jinja2 anaconda-client
+      conda install --yes conda-build jinja2 anaconda-client
```

Comment:
Forgot the unpinning in PR ( https://github.com/conda-forge/cmake-feedstock/pull/8 ). Have tacked it on to this PR. Basically activation scripts like that in the `toolchain` seem to produce faulty packages when there is a large differential between `conda` and `conda-build` as seen in this issue ( https://github.com/conda-forge/hdf5-feedstock/issues/23 ). While this problem seems to have gone away with newer `conda` and `conda-env` packages (for unclear reasons), we still try to unpin this just to be safe.